src/sbt-test/docker/file-permission/build.sbt

```diff
@@ -0,0 +1,96 @@
+lazy val checkDockerfile = taskKey[Unit]("")
```

Comment:
Very good test coverage :+1: 

Could you rename the `tasks` and `sbt` files to match what they test? e.g.

- `checkDockerfile` to `checkDockerfileDefaults`
- `checkDockerfile2` to `checkDockerfileWithPermissionsNone`

and also the sbt files `a.sbt` to `permissionsNone.sbt`. :smile:

Reply:
ok. implemented this in ef77500.